pkg/kn/commands/service/create.go

```diff
@@ -83,7 +83,16 @@ var create_example = `
   kn service create gitopstest --image knativesamples/helloworld --target=/user/knfiles/test.json
 
   # Create a service with profile
-  kn service create profiletest --image knativesamples/helloworld --profile istio`
+  kn service create profiletest --image knativesamples/helloworld --profile istio
+  
+  # Create a service with node selector (if feature flag is enabled here: https://knative.dev/docs/serving/configuration/feature-flags)
+  kn service create nodeselectortest --image knativesamples/helloworld --node-selector Disktype="ssd"
+
+  # Create a service with toleration (if feature flag is enabled here: https://knative.dev/docs/serving/configuration/feature-flags)
+  kn service create tolerationtest --image knativesamples/helloworld --toleration Key="node-role.kubernetes.io/master",Effect="NoSchedule",Operator="Equal",Value=""
+
+  # Create a service with node affinity (if feature flag is enabled here: https://knative.dev/docs/serving/configuration/feature-flags)
+  kn service create nodeaffinitytest --image knativesamples/helloworld --node-affinity Type="Required",Key="topology.kubernetes.io/zone",Operator="In",Values="antarctica-east1 antarctica-east2"`
```

Comment:
Preferably those docs link should point to exact feature anchor, e.g. `https://knative.dev/docs/serving/configuration/feature-flags/#kubernetes-toleration`.